.vinext/fonts/geist-8ac0455e797f/style.css

```
@@ -0,0 +1,45 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Geist';
+  font-style: normal;
+  font-weight: 100 900;
+  font-display: swap;
+  src: url(/Users/owainlewis/Code/github/owainlewis/contentflow/.vinext/fonts/geist-8ac0455e797f/geist-ff2310f5.woff2) format('woff2');
+  unicode-range: U+0460-052F, U+1C80-1C8A, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;
+}
+/* cyrillic */
+@font-face {
+  font-family: 'Geist';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)